com.unity.multiplayer.mlapi/Tests/Runtime/RpcPipelineTestComponent.cs

```diff
@@ -0,0 +1,170 @@
+using System.Collections.Generic;
+using UnityEngine;
+using MLAPI;
+using MLAPI.Messaging;
+
+namespace MLAPI.RuntimeTests
+{
+    public interface IMLAPIUnitTestObject
+    {
+        public bool IsTestComplete();
```

Comment:
> error CS0106: The modifier 'public' is not valid for this item

you can't specify access modifiers on interfaces, they're always going to be `public`

Reply:
Haha... oh! no! (don't even need that interface anymore)